src/metabase/db/liquibase.clj

```diff
@@ -31,6 +31,10 @@
 
 (set! *warn-on-reflection* true)
 
+(defonce ^{:doc "The set of Liquibase instances which potentially have taken locks by this process."}
```

Comment:
In which case there is more than one liquibase instance that held a lock?

Reply:
As mentioned in some of my other comments, this namespace is agnostic as to how its used. Theoretically you could have multiple "app dbs", for example when testing multiple drivers in the REPL.